src/css/homepage.css

```diff
@@ -89,3 +140,56 @@
 .homepage-shape-links-container > img {
     width: 10%;
 }
+
+/* Homepage theme park shape CSS & animation */
+
+.homepage-shape-links-container {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    padding: 2em;
+    background-image: linear-gradient(white, white 30%, white 40%, dodgerBlue 100%);
+}
+
+.homepage-shape-links-container > img {
+    width: 10%;
+    padding-bottom: 2em;
+
+}
+
+.homepage-shape-links-container-img{
+    display: flex;
+}
+
+.homepage-shape-links-container-img:hover{
```

Comment:
See my comments about it on the HTML, this should be moved to: `.homepage-shape-links-container > img:hover`